dandi/tests/test_download.py

```diff
@@ -147,6 +149,69 @@ def test_download_000027_resume(
     assert digester(str(nwb)) == digests
 
 
+@mark.skipif_no_network
+@pytest.mark.parametrize(
+    "dlmode", (DownloadExisting.OVERWRITE_DIFFERENT, DownloadExisting.REFRESH)
+)
+def test_download_000027_digest(
+    tmp_path: Path, dlmode: DownloadExisting, caplog: pytest.LogCaptureFixture
+) -> None:
+    from ..support.digests import Digester
```

Comment:
Move this import to the top of the file.

Reply:
got it. was just following existing test example:
https://github.com/dandi/dandi-cli/blob/b1a694e8f56222b21fda1400c311ad84985dfd00/dandi/tests/test_download.py#L61
https://github.com/dandi/dandi-cli/blob/b1a694e8f56222b21fda1400c311ad84985dfd00/dandi/tests/test_download.py#L127